cc_plugin_glider/glider_dac.py

```diff
@@ -41,6 +45,62 @@ class GliderCheck(BaseNCCheck):
         'Slocum Glider'
     }
 
+    def __init__(self):
+        # try to get the sea names table
+        ncei_base_table_url = 'https://gliders.ioos.us/ncei_authority_tables/'
+        # might refactor if the authority tables names change
+        table_type = {'project': 'projects.txt',
+                      'platform': 'platforms.txt',
+                      'instrument': 'instruments.txt',
+                      'institution': 'institutions.txt'}
+        # some top level attrs map to other things
+        var_remap = {'platform': 'id',
+                     'instrument': 'make_model'}
+
+        self.auth_tables = {}
+        for global_att_name, table_file in six.iteritems(table_type):
+            # instruments have to be handled specially since they aren't
+            # global attributes
+            table_loc = urljoin(ncei_base_table_url, table_file)
+            self.auth_tables[global_att_name] = GliderCheck.request_resource(table_loc, None,
```

Comment:
NB: loads NCEI tables at checker initialization time instead of making requests each time.